docs/Localiser/index.md

```diff
@@ -8,40 +8,11 @@ The following example is using strings found in the DayZ stringtable.
 The localiser class has overloaded the array assignment operator to allow setting of the string table keys and placing in other variables at the desired format.
 
 ```csharp
-CF_Localiser localiser = new CF_Localiser("STR_DATE_FORMAT");
+CF_Localiser localiser = new CF_Localiser("STR_DATE_FORMAT"); // "%a, %b %d"
```

Comment:
What is this supposed to be? a b d? Format uses %1 %2 etc

Reply:
It's not meant to work? That's what the dayz stringtable uses.

I haven't tested the example, just searched through the dayz stringtable to find a string with formatting and then made sure it compiled.

Reply:
Find a tested working example then :)